[2021-12-08 23:27:06,761] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-08T23:27:05.072663+00:00 [queued]>
[2021-12-08 23:27:06,795] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-08T23:27:05.072663+00:00 [queued]>
[2021-12-08 23:27:06,796] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 23:27:06,797] {taskinstance.py:1242} INFO - Starting attempt 1 of 3
[2021-12-08 23:27:06,797] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 23:27:06,829] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): check_imageFile> on 2021-12-08 23:27:05.072663+00:00
[2021-12-08 23:27:06,835] {standard_task_runner.py:52} INFO - Started process 6626 to run task
[2021-12-08 23:27:06,845] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'check_imageFile', 'manual__2021-12-08T23:27:05.072663+00:00', '--job-id', '627', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp226waskp', '--error-file', '/tmp/tmpap44y0bt']
[2021-12-08 23:27:06,847] {standard_task_runner.py:77} INFO - Job 627: Subtask check_imageFile
[2021-12-08 23:27:06,994] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-08T23:27:05.072663+00:00 [running]> on host b5fd0372bf6e
[2021-12-08 23:27:07,164] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=check_imageFile
AIRFLOW_CTX_EXECUTION_DATE=2021-12-08T23:27:05.072663+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-08T23:27:05.072663+00:00
[2021-12-08 23:27:07,168] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-08 23:27:07,179] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sha256sum /opt/***/official_data/Image.csv']
[2021-12-08 23:27:07,210] {subprocess.py:85} INFO - Output:
[2021-12-08 23:27:07,257] {subprocess.py:89} INFO - e891870a24e5a22c9520c5d417af923366083a3617288cafa3cc18e3fed77b11  /opt/***/official_data/Image.csv
[2021-12-08 23:27:07,258] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-08 23:27:07,384] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=check_imageFile, execution_date=20211208T232705, start_date=20211208T232706, end_date=20211208T232707
[2021-12-08 23:27:07,493] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-08 23:27:07,700] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
